drivers/kscan/Kconfig.npcx

```diff
@@ -0,0 +1,69 @@
+# NPCX Keyboard Scan driver configuration options
+
+# Copyright (c) 2022 Nuvoton Technology Corporation.
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig KSCAN_NPCX
+	bool "Nuvoton NPCX embedded controller (EC) Keyboard Scan (KSCAN) driver"
+	default y
+	depends on DT_HAS_NUVOTON_NPCX_KSCAN_ENABLED
+	select MULTITHREADING
+	help
+	  This option enables the KSCAN driver for NPCX family of
+	  processors.
+
+if  KSCAN_NPCX
+
+config KSCAN_NPCX_COLUMN_SIZE
+	int "Keyscan NPCX Column Size"
+	default 18
+	help
+	  Adjust the value to your keyboard columns. The maximum
+	  column size for the Nuvoton NPCX family is 18.
+
+config KSCAN_NPCX_ROW_SIZE
+	int "KSCAN_NPCX_ROW_SIZE"
+	default 8
+	help
+	  Adjust the value to your keyboard rows. The maximum
+	  row size for the Nuvoton NPCX family is 8.
+
+config KSCAN_NPCX_DEBOUNCE_DOWN
+	int "Keyscan NPCX Debounce Down"
+	default 10
+	help
+	  Determines the time in msecs for debouncing a key press.
+
+config KSCAN_NPCX_DEBOUNCE_UP
+	int "Keyscan NPCX Debounce Up"
+	default 20
+	help
+	  Determines the time in msecs for debouncing a key release.
+
+config KSCAN_NPCX_POLL_PERIOD
```

Comment:
This and the ones above should have a suffix for the unit `KSCAN_NPCX_POLL_PERIOD_MS` `KSCAN_NPCX_POLL_COL_OUTPUT_SETTLE_TIME_US` etc

Reply:
Done.